### DISCUSSION TITLE
Access Camera from Windows Share

### DISCUSSION THREAD

**Discussion by rspierenburg** [gtxaspec/wz_mini_hacks#527] [Q&A] [answered]
I was wondering if someone could help me.  I'm looking to either a) access the sd card from a windows computer without having to take the card out or b) have the camera save video to a windows share.  I have enable CIPS in the config, but I cannot access it from the windows machine.  Is there something else I need to be doing?  Any help would be greatly appreciated.

Thanks,
Rob

**Comment by bjs-pdx** [comment]
Interesting question and not really covered in the readme.  My assumption is that you can use cifs to mount a Windows share on the camera using the fstab file (ala linux) and then you can store things on it.

SCP is another option:  https://github.com/gtxaspec/wz_mini_hacks/discussions/135

**Comment by rspierenburg** [comment]
Is it possible to do it the other way around and have the camera connect to a Shared directory on the Windows Server?  I guess I'm not 100% sure on what CIPS in the config does exactly.

**Reply by bjs-pdx** [reply, marked as answer]
That's what I said, " use cifs to mount a Windows share on the camera " but probably wasn't very clear about it.  You'd need to see if the camera has an 'etc/fstab' file and setup appropriate entries there.  Here are the contents from /SD_ROOT/wz_mini/etc/fstab:

```
# <file system>	<mount pt>	<type>	<options>	<dump>	<pass>
proc		/proc		proc	defaults	0	0
devpts		/dev/pts	devpts	defaults,gid=5,mode=620	0	0
#tmpfs		/dev/shm	tmpfs	mode=0777	0	0
tmpfs		/tmp		tmpfs	mode=1777	0	0
tmpfs		/run		tmpfs	mode=0755,nosuid,nodev	0	0
sysfs		/sys		sysfs	defaults	0	0
media		/media		tmpfs	mode=1777	0	0
tmpfs           /opt/wz_mini/tmp            tmpfs   mode=1777       0       0
```

**Reply by rspierenburg** [reply]
Great thanks, I think I just need to learn more about fstab.

**Reply by rspierenburg** [reply]
So I was able to mount the share, many thanks for that, but now I need to find a way to change the save location from the sd card to the share.

**Reply by bjs-pdx** [reply]
Have you looked at the readme file?  

https://github.com/gtxaspec/wz_mini_hacks#mp4_write

That's a start.

This is a discussion on using NFS for your purposes...perhaps similar for cifs?

https://github.com/gtxaspec/wz_mini_hacks/discussions/107

**Reply by rspierenburg** [reply]
Yes, I checked the readme but it doesn't explain how.

However the thread you posted looks promising.  Thanks for all your help.

**Reply by rspierenburg** [reply]
Actually...I had the fstab mount the cifs to the MMC folder, but I since changed it to the MMC/Record folder and it seems to populate now.  Thanks again.

**Reply by tdashmike** [reply]
> So I was able to mount the share, many thanks for that, but now I need to find a way to change the save location from the sd card to the share.

Can you share your fstab entry. Mine will not mount during boot but if I run mount-a after it's booted it mounts it.

//192.168.x.x/Security/B /media/mmc/record cifs username=a,password=b,vers=2.0 0 0

Thank you!
